.github/workflows/main.yaml

```diff
@@ -30,8 +30,10 @@ jobs:
             ./Containerfile
           image: ghcr.io/led0nk/images
           oci: false
+          build-args: |
+            BASE_IMAGE="registry.fedoraproject.org/fedora-silverblue"
+            FEDORA_VERSION="latest"
```

Comment:
Thats fedora rawhide?

Reply:
the up to date commit is not pushed yet, switched over to `quay.io/fedora/fedora:40`